apps/backend/src/app/descart/descart.service.ts

```diff
@@ -208,10 +210,11 @@ export class DescartService {
       .addSelect('COUNT(users.id)', 'favorite')
       .addSelect('product.name', 'productName')
       .addSelect('manufacturer.name', 'manufacturerName')
+      .addSelect('category.name', 'categoryName')
       .addSelect('product.imageUrl', 'imageUrl')
       .offset(Number(page) * Number(pageSize))
       .limit(Number(pageSize));
-    if (!(search && search.length != 0) && !(favorite == 'true')) {
+    if (!(search && search.length != 0) && !(favorite == 'true') && !(catIds && catIds.length > 0)) {
```

Comment:
Just making sure, the front end will always send all three of these? Otherwise it would do none of the following logic I think.

Reply:
yes it always sends all 3, I've tested by hardcoding category ids that it works properly